### PR TITLE
Fix remote repository URL in .shed.yml

### PR DESCRIPTION
**Pull request by davebx**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [x] - This PR updates an existing tool or tool collection

### CODE REVIEW

**Review by mvdbeek — APPROVED** [review]
Thanks @davebx!

**Comment by nsoranzo** [comment]
There are some broken links referenced from the tool, see https://github.com/galaxyproject/tools-iuc/pull/3439/checks?check_run_id=1742791234

File: tools/gatk4/gatk4_Mutect2.xml [hunk 1]
```diff
@@ -702,11 +702,8 @@ with the assembly-based machinery of
 `HaplotypeCaller <https://www.broadinstitute.org/gatk/documentation/tooldocs/org_broadinstitute_gatk_tools_walkers_haplotypecaller_HaplotypeCaller.php>`__.
```
Comment (nsoranzo) [inline tools/gatk4/gatk4_Mutect2.xml:702]:
```suggestion
`HaplotypeCaller <https://gatk.broadinstitute.org/hc/en-us/articles/360035531412-HaplotypeCaller-in-a-nutshell>`__.
```

**Comment by nsoranzo** [comment]
Linting of URLs fails because CloudFlare is configured to block scrapers, https://github.com/galaxyproject/planemo/pull/1134 will workaround this.

**Pull request merged**